src/main/resources/templates/dashboard.twig

```diff
@@ -39,9 +44,18 @@
                     </script>
                     {% endblock %}
                     <p>Select a course from the course list, or create a course.</p>
-                    <button type="button" data-target="#courseForm" class="btn btn-primary" data-toggle="collapse"
-                            aria-expanded="false" aria-controls="courseForm">Create Course
-                    </button>
+                    {% if courseCreationError %}
```

Comment:
The branches of this if are identical, is it the aria-expanded value supposed to be changed?

Reply:
Nope, we thought we were going to have to do something here, and forgot to revert this part.